examples/seek_viewer.cpp

```diff
@@ -164,17 +183,54 @@ int main(int argc, char** argv)
         }
 
         // Retrieve frame from seek and process
-        process_frame(seekframe, outframe, scale, colormap, rotate);
+        process_frame(seekframe, outframe, scale, colormap, rotate, normalize, brightness);
 
         if (output == "window") {
             imshow("SeekThermal", outframe);
-            char c = waitKey(10);
-            if (c == 's') {
+		namedWindow("SeekThermal");
+		moveWindow("SeekThermal", x,y);
```

Comment:
This blocks the user from moving the window using their window manager. That doesn't seem reasonable.